### PR TITLE
Add release notes for 5.14.2

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds release notes draft for 5.14.2

## Motivation and Context
5.14.2 planned for Nov 3

## Review Instructions
- Release info gathered from https://github.com/sensu/sensu-go/commits/release/5.14

### CODE REVIEW

File: content/sensu-go/5.15/reference/entities.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Proxy entities allow Sensu to monitor external resources on systems where a Sens
 
 ## Usage limits
 
-This version of Sensu has no functional limitations based on entity count. If your Sensu instance includes over 1,000 entities, contact us to learn about [license-activated features][9] designed for monitoring at scale. See [Discourse][10] for more information about our usage policy. 
+Sensu's free entity limit is 100 entities. All [license-activated features][9] are available for free in the packaged Sensu Go distribution up to an entity limit of 100. If your Sensu instance includes more than 100 entities, [contact us][10] to learn how to upgrade your installation and increase your limit. See [the announcement on our blog][11] for more information about our usage policy.
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/entities.md]:
```suggestion
This version of Sensu has no functional limitations based on entity count. If your Sensu instance includes over 1,000 entities, contact us to learn about [license-activated features][9] designed for monitoring at scale. See [Discourse][10] for more information about our usage policy. 
```

File: content/sensu-go/5.15/reference/entities.md [hunk 2]
```diff
@@ -803,9 +803,6 @@ spec:
 [api-filter]: ../../api/overview#filtering
 [sensuctl-filter]: ../../sensuctl/reference#filtering
 [9]: ../../getting-started/enterprise
-[10]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156
-[11]: ../checks#round-robin-checks
-[12]: ../../guides/monitor-external-resources/#using-a-proxy-entity-to-monitor-a-website
-[13]: ../checks/#proxy-requests
-[14]: ../../guides/monitor-external-resources/
-[15]: #proxy-entities
+[10]: https://sensu.io/contact
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/entities.md]:
```suggestion
```

File: content/sensu-go/5.15/reference/entities.md [hunk 3]
```diff
@@ -803,9 +803,5 @@ spec:
 [api-filter]: ../../api/overview#filtering
 [sensuctl-filter]: ../../sensuctl/reference#filtering
 [9]: ../../getting-started/enterprise
-[10]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156
-[11]: ../checks#round-robin-checks
-[12]: ../../guides/monitor-external-resources/#using-a-proxy-entity-to-monitor-a-website
-[13]: ../checks/#proxy-requests
-[14]: ../../guides/monitor-external-resources/
-[15]: #proxy-entities
+[11]: https://blog.sensu.io/one-year-of-sensu-go
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/entities.md]:
```suggestion
```

File: content/sensu-go/5.15/reference/entities.md [hunk 4]
```diff
@@ -803,9 +803,4 @@ spec:
 [api-filter]: ../../api/overview#filtering
 [sensuctl-filter]: ../../sensuctl/reference#filtering
 [9]: ../../getting-started/enterprise
-[10]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/entities.md]: [10]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156

File: content/sensu-go/5.15/reference/entities.md [hunk 5]
```diff
@@ -803,9 +803,4 @@ spec:
 [api-filter]: ../../api/overview#filtering
 [sensuctl-filter]: ../../sensuctl/reference#filtering
 [9]: ../../getting-started/enterprise
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/entities.md]:
```suggestion
[9]: ../../getting-started/enterprise
[10]: https://discourse.sensu.io/t/introducing-usage-limits-in-the-sensu-go-free-tier/1156
[11]: ../checks#round-robin-checks
[12]: ../../guides/monitor-external-resources/#using-a-proxy-entity-to-monitor-a-website
[13]: ../checks/#proxy-requests
[14]: ../../guides/monitor-external-resources/
[15]: #proxy-entities
```

File: content/sensu-go/5.15/reference/entities.md [hunk 6]
```diff
@@ -809,3 +809,4 @@ spec:
 [13]: ../checks/#proxy-requests
 [14]: ../../guides/monitor-external-resources/
 [15]: #proxy-entities
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/entities.md]:
```suggestion
```

File: content/sensu-go/5.14/release-notes.md [hunk 7]
```diff
@@ -42,6 +43,28 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
 
+## 5.14.2 release notes
+
+**November 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.2, is now available for download. This release includes an etcd upgrade, fixes that improve stability and performance, and a Sensu Go package for CentOS 8.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.14.2.
+
+**NEW FEATURES**
+
+- ([Licensed tier][79]) Added build package for CentOS 8 (`el/8`)
```
Comment (hillaryfraley) [inline content/sensu-go/5.14/release-notes.md]:
```suggestion
- ([Licensed tier][79]) Added build package for CentOS 8 (`el/8`).
```

File: content/sensu-go/5.14/release-notes.md [hunk 8]
```diff
@@ -42,6 +43,28 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.14.2 release notes
+
+**November 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.2, is now available for download. This release includes an etcd upgrade, fixes that improve stability and performance, and a Sensu Go package for CentOS 8.
```
Comment (bananabele) [inline content/sensu-go/5.14/release-notes.md]: Date should be Nov 4 📆

Comment (hillaryfraley) [reply content/sensu-go/5.14/release-notes.md]: Thank you!

Comment (hillaryfraley) [inline content/sensu-go/5.14/release-notes.md]:
```suggestion
**November 4, 2019** &mdash; The latest release of Sensu Go, version 5.14.2, is now available for download. This release includes an etcd upgrade, fixes that improve stability and performance, and a Sensu Go package for CentOS 8.
```

**Review by echlebek — APPROVED** [review]
:+1:

**Pull request merged**